src/tsickle.ts

```diff
@@ -867,7 +867,8 @@ class Annotator extends ClosureRewriter {
         // If it has a symbol, it's actually a regular declared property.
         if (!quotedPropSym) return false;
         const declarationHasQuotes =
-            !quotedPropSym.declarations || quotedPropSym.declarations.some(decl => {
+            !quotedPropSym.declarations || quotedPropSym.declarations.some(d => {
+              const decl = d as ts.NamedDeclaration;
               if (!decl.name) return false;
```

Comment:
the !decl.name looks a bit as if this was handling `NamedDeclaration`s and `Declaration`s. Do we have a better way to runtime test if this is named or unnamed? `if (typeof decl.name !== undefined)` or so?